### ISSUE TITLE
(Bug) Playlists on Soundnode and Soundcloud are different

### ISSUE THREAD

**Issue by DrewParsons** [Soundnode/soundnode-app#878]
Soundnode is missing songs on my playlist that I have on Soundcloud.  Some time in June I was adding/subtracting songs from my playlist using Soundnode and It deleted about 20 of my songs on my large playlist and some on my other playlists.  Super weird.  I've uninstalled/reinstalled a couple of times and also cleaned local storage within the app's settings in hopes that Soundnode and my Soundcloud web playlist would change to be the same with no luck.  

my "Rap" playlist on Soundnode with only 3 songs.
<img width="986" alt="screen shot 2016-09-21 at 9 58 48 pm" src="https://cloud.githubusercontent.com/assets/13970454/18735888/1b150ade-8047-11e6-9ab2-469937f7361b.png">

The same playlist on Soundcloud.
![screen shot 2016-09-21 at 9 59 17 pm](https://cloud.githubusercontent.com/assets/13970454/18735898/31a87484-8047-11e6-9ad1-57c89490ad80.png)

**Comment by DrewParsons** [comment]
(UPDATE:) 
After further inspection, it seems that Soundnode doesn't like some Artists. (I know that sounds crazy) I looked at some of the songs that were missing in one of my playlists, and started adding other songs from the same artist to the end of the playlist.  After adding a few songs, I compared.  

This is my playlist on Soundcloud, and I added the last 3 songs as a test.  
<img width="1252" alt="screen shot 2016-09-21 at 10 14 25 pm" src="https://cloud.githubusercontent.com/assets/13970454/18736051/d71ec05c-8048-11e6-9279-f068b6ba8991.png">

After Re-opening SoundNode and looking at the same playlist, we see that the Prince Fox song and the Seven Lions song are missing, after I had just added them, but the Daruma song was added perfectly. 
<img width="1177" alt="screen shot 2016-09-21 at 10 16 03 pm" src="https://cloud.githubusercontent.com/assets/13970454/18736060/01f5fe9e-8049-11e6-8cc9-8b2f6d057bdd.png">

All of the songs that I had on this playlist from those two artists (along with a bunch of others) are all missing as well.  Help!
